sections/errorhandling/testingerrorflows.spanish.md

```diff
@@ -1,38 +1,81 @@
-# Testear flujos de error mediante tu framework favorito de testing
+# Prueba flujos de error mediante tu framework favorito de testing
 
 ### Párrafo de explicación
 
-Testear rutas de ejecución "felices" no es mejor que testear errores. Un buen coverage de código exige testear rutas de ejecución adversas. De lo contrario, no existe ninguna seguridad de que las excepciones se capturen y gestionen correctamente. Cada framework de testing unitario, como Mocha y Chai, admite testing de excepciones (códigos de ejemplo a continuación). Si te parece tedioso ir probando cada función y excepción internas, podrías quedarte solo con el testing de errores REST API HTTP.
+Probar rutas de ejecución "felices" no es mejor que probar errores. Una buena cobertura de código exige testear rutas de ejecución adversas. De lo contrario, no existe ninguna seguridad de que las excepciones se capturen y gestionen correctamente. Cada framework de pruebas unitario, como [Mocha](https://mochajs.org/) y [Chai](http://chaijs.com/), admite pruebas de excepciones (códigos de ejemplo a continuación). Si te parece tedioso ir probando cada función y excepción internas, podrías quedarte solo con las pruebas de errores REST API HTTP.
```

Comment:
test and derivates untranslated.
Also added the links to the libraries that are being mentioned